src/gordon_gcp/schema/schemas/audit-log.schema.json

```diff
@@ -23,7 +23,7 @@
                     "examples": [
                         "projects/a-project-id/zones/a-zone-name/instances/an-instance-name"
                     ],
-                    "pattern": "^projects/[0-9]+/zones/[a-z\\-0-9]+/instances/[a-z\\-0-9]+"
+                    "pattern": "^projects/([0-9]+|[a-z][a-z\\-0-9]{5,29})/zones/[a-z\\-0-9]+/instances/[a-z\\-0-9]+"
```

Comment:
I'd be willing to be more lenient for easier readability (and potentially for if GCP changes their requirements), like `[a-z\\-0-9]+`. I'm not sure we gain much by being so specific.

Reply:
I mean, legit.  But those aren't the same because of the initial letter.  Shrug.